crates/source_analyzer/src/uses.rs

```diff
@@ -1,11 +1,11 @@
 use std::collections::HashMap;
 
-use syn::{Ident, Item, UseTree};
+use syn::{File, Ident, Item, UseTree};
 
 pub type Uses = HashMap<Ident, Vec<Ident>>;
 
-pub fn uses_from_items(items: &[Item]) -> Uses {
-    items
+pub fn uses_from_file(file: &File) -> Uses {
+    file.items
```

Comment:
What was the reason to make this less generic?

Reply:
Undone, also removed the unnecessary trait below.